tests/Unit/Pdf/PdfPrintOptionsTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace EMS\CommonBundle\Tests\Unit\PdfPrintOptionsTest;
+
+use EMS\CommonBundle\Service\Pdf\PdfPrintOptions;
+use PHPUnit\Framework\TestCase;
+
+class PdfPrintOptionsTest extends TestCase
```

Comment:
The PdfPrintOptions class is also a "model" class. We defined that we would not test Getters/Setters.